src/components/Header.js

```diff
@@ -49,21 +68,30 @@ class Header extends React.Component {
         style={styles.navbar}
         expand="sm"
       >
-        <Link route="index" passHref>
-          <NavbarBrand>Queues@Illinois</NavbarBrand>
-        </Link>
-        <NavbarToggler onClick={() => this.toggle()} />
+        {brandLink}
         {user && (
-          <Collapse isOpen={this.state.isOpen} navbar>
-            <Nav navbar className="ml-auto">
-              <Link route="userSettings" passHref>
-                <NavLink className="navbar-text">
-                  <FontAwesomeIcon icon={faUser} className="mr-2" />
-                  {userName}
-                </NavLink>
-              </Link>
-            </Nav>
-          </Collapse>
+          <Fragment>
+            <NavbarToggler onClick={() => this.toggle()} />
+            <Collapse isOpen={this.state.isOpen} navbar>
+              <Nav navbar className="ml-auto">
+                <Link route="userSettings" passHref>
+                  <NavLink className="navbar-text mr-3">
+                    <FontAwesomeIcon icon={faUser} className="mr-2" />
+                    {userName}
+                  </NavLink>
+                </Link>
+                <NavItem>
+                  <Button
+                    color="secondary"
+                    type="button"
+                    onClick={() => { window.location = logoutRoute}}
```

Comment:
I think we should use `Router.pushRouter` here to keep things consistent, unless there's a reason for using `window.location`.

Reply:
I'm generally trying to keep `Router.*` reserved for things that are actually routable client-side - when logging out, we can't do that.

Reply:
Although, an interesting option would be hitting the logout link in the background so we don't need to land the user on the Shib logout page... thoughts?

Reply:
Makes sense, I think either way is fine.